addons/udes_stock/README.md

```diff
@@ -259,16 +259,14 @@ Pick3      | Pick001-02 | Pick001 | Pick1  |        -         |
 Pick4      | Pick001-02 | Pick001 | Pick2  |        -         |
 Pick5      | Pick001-02 | Pick001 | Pick2  |        -         |
 
-                Original Picking (Pick001)
-                        |
-        ----------------------------------------
-        |                                       |
-    pick1  (Pick001-001)                      pick 2  (Pick001-001)
-        |                                       |
-        |                           -------------------------
-        |                           |                       |
-    pick3  (Pick001-002)        pick4  (Pick001-002)        pick5  (Pick001-002)
-
+```mermaid
+graph TD;
+    A["Original Picking (Pick001)"] --&gt; B["pick1 (Pick001-001)"];
```

Comment:
doesn't seem to like `&gt;`, need to use `>` instead

Reply:
yeah just saw that lol- although the copy and pasta rendered in mine locally ok

Reply:
that seesm to be ok.
Never used mermaid, thanks. I do like the impact of `------------------------------------->` has on the graph as well :)

Reply:
yeah you can do some pretty wacky stuff with it! only tried basic stuff though